runtime/interpreter/interpreter_tracing_test.go

```diff
@@ -0,0 +1,203 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright 2022 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package interpreter_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/cadence/runtime/interpreter"
+	"github.com/onflow/cadence/runtime/sema"
+	"github.com/onflow/cadence/runtime/tests/utils"
+)
+
+func TestInterpreterTracing(t *testing.T) {
+
+	t.Parallel()
+
+	t.Run("array tracing", func(t *testing.T) {
+		storage := interpreter.NewInMemoryStorage()
+
+		traceOps := make([]string, 0)
+		inter, err := interpreter.NewInterpreter(
+			&interpreter.Program{},
+			utils.TestLocation,
+			interpreter.WithOnRecordTraceHandler(
+				func(inter *interpreter.Interpreter,
+					operationName string,
+					duration time.Duration,
+					logs []opentracing.LogRecord) {
+					traceOps = append(traceOps, operationName)
+				},
+			),
+			interpreter.WithStorage(storage),
+			interpreter.WithTracingEnabled(true),
+		)
+		require.NoError(t, err)
```

Comment:
This setup code is repeated in each test. Consider defining a helper function for it

Reply:
added a helper function